release-steps.sh

```diff
@@ -9,8 +9,7 @@ python ./manage.py migrate --no-input
 python ./manage.py block_inventory
 
 # Wagtail translations
-python ./manage.py sync_page_translation_fields
-python ./manage.py update_translation_fields
+# TODO: May need to add in auto syncing using wagtail-localize in the future.
```

Comment:
I'm purposely leaving this in here in case we need to do some work around this later for auto syncing translations. I'm not sure what this will include yet, so I wrote a TODO in the code as a way to persist the change. We could throw all our todos in a ticket at the end of the sprint for "code cleanup" if it comes to that.